api/src/services/assets.ts

```diff
@@ -65,7 +68,7 @@ export class AssetsService {
 			await this.authorizationService.checkAccess('read', 'directus_files', id);
 		}
 
-		const file = (await this.knex.select('*').from('directus_files').where({ id }).first()) as File;
+		const file = (await this.filesService.readOne(id, { limit: 1 })) as File;
```

Comment:
```suggestion
		const file = (await this.filesService.readOne(id, { limit: 1 })) as File | undefined;
```

Looks like the FilesService types are a bit whack (possibly inherited from ItemsService) we'll want to fix that at some point.